intranet/apps/emerg/views.py

```diff
@@ -68,12 +68,30 @@ def check_emerg():
         title = safe_fcps_emerg_html(title, settings.FCPS_EMERGENCY_PAGE)
         body = safe_fcps_emerg_html(body, settings.FCPS_EMERGENCY_PAGE)
 
-        message = "<h3>{}: </h3>{}".format(title, body)
-        message = message.strip()
-    else:
-        status = False
+        announcements.append({"title": title, "body": body})
+
+    try:
+        r = requests.get(settings.CSL_STATUS_PAGE, timeout=timeout)
+    except requests.exceptions.Timeout:
+        pass
+
+    try:
+        csl_status = json.loads(r.text)
+    except json.decoder.JSONDecodeError:
+        return False, None
+
+    for system in csl_status['systems']:
```

Comment:
I think that with the code that's currently here, the status isn't updated if there's a  CSL issue, so you'll never see a CSL issue unless there's an FCPS emergency announcement as well. Having `status` update whenever there's a CSL issue will fix that. Also, we probably don't want informational updates to be in the yellow (e.g. current Borg cluster maintenance, so we can filter that out), and we shouldn't have multiple instances of the same thing in the `announcements` list (the JSON has the same issue under each affected system).

With all that said, here's how I worked around all of this:
```py
status = False
    for system in csl_status['systems']:
        if system['status'] != 'ok':
            status = True
            issues = system['unresolvedIssues']
            for issue in issues:
                desc = requests.get(issue['permalink']).text
                soup = BeautifulSoup(desc, "html.parser")
                desc = soup.find_all('p')[1].text
                val = {"title": issue['title'], "body": desc}
                if val not in announcements and issue['severity'] != 'notice':
                    announcements.append(val)
```